internal/manager/config.go

```diff
@@ -89,6 +89,8 @@ type Config struct {
 
 // onOffUsage is used to indicate what textual options are used to enable or disable a feature.
 const onOffUsage = "Can be one of [enabled, disabled]."
+const onOffAutoIngressUsage = "Can be one of [enabled, disabled, auto]. If multiple ingress API versions are set to 'auto', then the newest out of these will be used."
+const onOffAutoCRDUsage = "Can be one of [enabled, disabled, auto]. 'auto' means that enablement will depend on the detected presence of the appropriate CRD in the cluster."
```

Comment:
If both disabled and auto, which one win?

Reply:
#1638 removed the distinction.